spec/models/claim_spec.rb

```diff
@@ -654,34 +664,34 @@
 
   describe "#amendable?" do
     it "returns false for a claim that hasn’t been submitted" do
-      claim = build(:claim, :submittable)
+      claim = create(:claim, :submittable)
```

Comment:
Does this one need to be changed to `create`?

Reply:
as above, if we change the factory we can hopefully leave these as they are.